packages/radio-group/test/radio-button.test.js

```diff
@@ -95,6 +95,14 @@ describe('radio-button', () => {
       radio.click();
       expect(spy.called).to.be.false;
     });
+
+    it('should focus on input click if not focused', async () => {
+      const rect = input.getBoundingClientRect();
+      const middleX = Math.floor(rect.x + rect.width / 2);
+      const middleY = Math.floor(rect.y + rect.height / 2);
+      await sendMouse({ type: 'click', position: [middleX, middleY] });
```

Comment:
Remember to reset the mouse position to avoid side effects on the next tests.

Reply:
Thanks, done.